src/gretel_trainer/relational/json.py

```diff
@@ -129,6 +127,34 @@ def make_suffix(s):
     return hashlib.sha256(s.encode("utf-8")).hexdigest()[:10]
 
 
+def jsonencode(df: pd.DataFrame, cols: list[str]) -> pd.DataFrame:
```

Comment:
This is a great find on writing dataframes that contain dicts/lists as CSV and preserving its "JSON status".